### ISSUE TITLE
windows state.user not adding users to Remote Desktop Users

### ISSUE THREAD

**Issue by Inveracity** [saltstack/salt#27865]
Heya

the "Remote Desktop Users" group is ignored when I add new users.
I've tried with and without quotes around it.
The minion is a Windows Server 2012 R2.

command:
```
# salt '*' state.apply standard.users
```
file:
/srv/salt/standard/users.sls
```
{% for usr in pillar['users'] %}
{{usr}}:
  user.present:
    - password: supersecretpassword
    - groups: 
      - Users
      - Administrators
      - Remote Desktop Users
```
--------------------
Salt-Master versions-report
```
Salt Version:
           Salt: 2015.8.0

Dependency Versions:
         Jinja2: 2.7.2
       M2Crypto: 0.21.1
           Mako: 0.9.1
         PyYAML: 3.10
          PyZMQ: 14.4.0
         Python: 2.7.6 (default, Jun 22 2015, 17:58:13)
           RAET: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.0.4
           cffi: Not Installed
       cherrypy: 3.2.2
       dateutil: 1.5
          gitdb: 0.5.4
      gitpython: Not Installed
          ioflo: Not Installed
        libnacl: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.3.0
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
         pygit2: Not Installed
   python-gnupg: Not Installed
          smmap: 0.8.2
        timelib: Not Installed

System Versions:
           dist: Ubuntu 14.04 trusty
        machine: x86_64
        release: 3.13.0-32-generic
         system: Ubuntu 14.04 trusty

```

Salt-Minion versions-report
```
           Salt: 2015.5.2
         Python: 2.7.8 (default, Jun 30 2014, 16:08:48) [MSC v.1500 64 bit (AMD64)]
         Jinja2: 2.7.3
       M2Crypto: 0.21.1
 msgpack-python: 0.4.5
   msgpack-pure: Not Installed
       pycrypto: 2.6
        libnacl: Not Installed
         PyYAML: 3.11
          ioflo: Not Installed
          PyZMQ: 14.6.0
           RAET: Not Installed
            ZMQ: 4.0.5
           Mako: Not Installed
```

**Comment by jfindlay** [comment]
@Inveracity, thanks for the report.

**Comment by twangboy** [comment]
@Inveracity I can't reproduce this on a 2015.8 minion. There was a problem with 2015.5 minions dealing with groups with spaces. That has been fixed in 2015.8.

**Comment by Inveracity** [comment]
:+1: excellent thanks
